x-pack/plugins/security_solution/public/management/hooks/endpoint/use_with_show_endpoint_responder.tsx

```diff
@@ -55,6 +55,8 @@ export const useWithShowEndpointResponder = (): ShowEndpointResponseActionsConso
             PageTitleComponent: () => <>{RESPONDER_PAGE_TITLE}</>,
             PageBodyComponent: () => <OfflineCallout endpointId={endpointAgentId} />,
             ActionComponents: [ActionLogButton],
+            // hide the console Done/close button
+            showCloseButton: false,
```

Comment:
Could we set this to `false` by default to avoid adding this extra prop here?

Reply:
959e25bcb3464bb605a51548a35f768bc40a6354